Vagrantfile

```diff
@@ -30,7 +30,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.synced_folder ".", "/vagrant", type: "virtualbox"
 
   config.vm.provider :virtualbox do |vb|
-    vb.customize ["modifyvm", :id, "--memory", "512"]
+    vb.customize ["modifyvm", :id, "--memory", "1512"]
+    vb.cpus = 2
```

Comment:
Why add CPU's?

Reply:
Had a few occurrences of startup timing out when testing and figured more CPUs would help given how resource intensive puppetserver can be.

Reply:
This can slow things down with virtualization as now there needs to be two free cpu's available at the same time. If your system only has one at that moment, then the hypervisor waits until both are free.

Reply:
Removed cpus change.